ci-operator/step-registry/openstack/provision/sriov-networknodepolicy/openstack-provision-sriov-networknodepolicy-commands.sh

```diff
@@ -54,6 +54,27 @@ then
 	source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
+# Even if the pods are ready, we need to wait for the webhook server to be
+# actually started, which usually takes a few seconds.
+for _ in $(seq 1 30); do
+    WEBHOOK_NAME=$(oc get validatingwebhookconfigurations.admissionregistration.k8s.io  sriov-operator-webhook-config -o jsonpath='{.metadata.name}')
```

Comment:
there is an extra "space" after `validatingwebhookconfigurations.admissionregistration.k8s.io`